channels/smartcard/client/smartcard_pack.c

```diff
@@ -2074,7 +2074,7 @@ LONG smartcard_pack_list_reader_groups_return(SMARTCARD_DEVICE* smartcard, wStre
 {
 	LONG status;
 	DWORD cBytes = ret->cBytes;
-	DWORD index = 0;
+	UINT32 index = 0;
```

Comment:
This should not be necessary, `UINT32` and `DWORD` are both defined to the same 32bit unsigned integer type, if not then something in winpr is missing for MinGW